### PR TITLE
Fix log broadcaster force flag

### PR DESCRIPTION
**Pull request by Nic0S**
"force" was used in one spot, and "force-broadcast" in another
preventing the flag from being propagated. Fix it.

### CODE REVIEW

**Review by jmank88 — DISMISSED** [review]

File: core/chains/evm/log/broadcaster.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func (b *broadcaster) Start(context.Context) error {
 
 // ReplayFromBlock implements the Broadcaster interface.
 func (b *broadcaster) ReplayFromBlock(number int64, forceBroadcast bool) {
-	b.logger.Infof("Replay requested from block number: %v", number)
+	b.logger.Infof("Replay requested from block number: %v, force: %t", number, forceBroadcast)
```
Comment (jmank88) [inline core/chains/evm/log/broadcaster.go:191]: nit/ could use `Infow`

Comment (Nic0S) [reply core/chains/evm/log/broadcaster.go]: SG, done

File: core/chains/evm/log/broadcaster.go [hunk 2]
```diff
@@ -188,7 +188,7 @@ func (b *broadcaster) Start(context.Context) error {
 
 // ReplayFromBlock implements the Broadcaster interface.
 func (b *broadcaster) ReplayFromBlock(number int64, forceBroadcast bool) {
-	b.logger.Infof("Replay requested from block number: %v", number)
+	b.logger.Infow("Replay requested", "block number", number, "force", forceBroadcast)
```
Comment (jmank88) [inline core/chains/evm/log/broadcaster.go:191]: It looks like we use the key `"blockNum"` a lot? :shrug: 
```suggestion
	b.logger.Infow("Replay requested", "blockNum", number, "force", forceBroadcast)
```

**Review by jmank88 — APPROVED** [review]

**Comment by jmank88** [comment]
Related? https://app.shortcut.com/chainlinklabs/story/32708/cli-tests-do-not-cover-cli-flag-type-validation

**Comment by Nic0S** [comment]
Yeah I think this would have been caught with that change

**Pull request merged**